pkg/virt-handler/container-disk/mount.go

```diff
@@ -34,6 +36,12 @@ const (
 	failedUnmountFmt         = "failed to unmount containerDisk %v: %v : %v"
 )
 
+var (
+	ErrChecksumMissing           = errors.New("missing checksum")
+	ErrChecksumMismatch          = errors.New("checksum mismatch")
+	ErrChecksumDiskContainerGone = errors.New("disk container is gone")
```

Comment:
This error is probably not related to checksumming. I would say it is a common one since it is used e.g. in
```go
	sock, err := m.socketPathGetter(vmi, volumeIndex)
	if err != nil {
		return nil, ErrChecksumDiskContainerGone
	}
```
Suggest to rename it:
```suggestion
	ErrDiskContainerGone = errors.New("disk container is gone")
```